vpr/src/device/rr_graph_view.h

```diff
@@ -249,6 +249,26 @@ class RRGraphView {
         return node_storage_.node_side_string(node);
     }
 
+    /** @brief Get the ptc num of a routing resource node. This function is inlined for runtime optimization. */
```

Comment:
The ``node_ptc_num()`` requires additional comments to guide developers. Can you adapt the text in RRGraphBuilder in this context? 

https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/bae983f4e018a96b92a8a0679c27e75d528e78b3/vpr/src/device/rr_graph_builder.h#L72-L99